src/styles/main.scss

```diff
@@ -0,0 +1,49 @@
+@import "../styles/utils/_variables.scss";
+@import "../styles/utils/resets";
+
+.clock {
```

Comment:
check this checklist point
<img width="653" alt="image" src="https://user-images.githubusercontent.com/102411130/213397690-fdad9044-e28b-4a31-8b29-c4ae463cbcf8.png">